[2021-12-22 17:29:15,277] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.super_resolution manual__2021-12-22T07:55:18.519597+00:00 [queued]>
[2021-12-22 17:29:15,281] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.super_resolution manual__2021-12-22T07:55:18.519597+00:00 [queued]>
[2021-12-22 17:29:15,281] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:29:15,281] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:29:15,281] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:29:15,289] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): super_resolution> on 2021-12-22 07:55:18.519597+00:00
[2021-12-22 17:29:15,291] {standard_task_runner.py:52} INFO - Started process 24112 to run task
[2021-12-22 17:29:15,293] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'super_resolution', 'manual__2021-12-22T07:55:18.519597+00:00', '--job-id', '106', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpnjaqh911', '--error-file', '/tmp/tmpmja0a_95']
[2021-12-22 17:29:15,293] {standard_task_runner.py:77} INFO - Job 106: Subtask super_resolution
[2021-12-22 17:29:15,311] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.super_resolution manual__2021-12-22T07:55:18.519597+00:00 [running]> on host myamya
[2021-12-22 17:29:15,331] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=super_resolution
AIRFLOW_CTX_EXECUTION_DATE=2021-12-22T07:55:18.519597+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-22T07:55:18.519597+00:00
[2021-12-22 17:29:15,332] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:29:15,332] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sure:0.3 /bin/bash /home/myamya/project/init.sh ']
[2021-12-22 17:29:15,336] {subprocess.py:85} INFO - Output:
[2021-12-22 17:29:16,930] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
[2021-12-22 17:29:16,931] {subprocess.py:89} INFO -   return torch._C._cuda_getDeviceCount() > 0
[2021-12-22 17:29:17,939] {subprocess.py:89} INFO - Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.1.0/detection_Resnet50_Final.pth" to /opt/conda/lib/python3.8/site-packages/facexlib/weights/detection_Resnet50_Final.pth
[2021-12-22 17:29:17,939] {subprocess.py:89} INFO - 
[2021-12-22 17:29:20,815] {subprocess.py:89} INFO -   0% 0.00/104M [00:00<?, ?B/s]  0% 200k/104M [00:00<00:53, 2.03MB/s]  1% 1.13M/104M [00:00<00:40, 2.67MB/s]  4% 4.36M/104M [00:00<00:28, 3.66MB/s]  8% 7.96M/104M [00:00<00:20, 5.02MB/s] 12% 12.4M/104M [00:00<00:14, 6.80MB/s] 17% 17.8M/104M [00:00<00:09, 9.24MB/s] 22% 23.2M/104M [00:00<00:06, 12.3MB/s] 26% 27.1M/104M [00:00<00:05, 15.4MB/s] 31% 32.5M/104M [00:00<00:03, 19.7MB/s] 36% 37.9M/104M [00:01<00:02, 24.5MB/s] 41% 42.6M/104M [00:01<00:02, 27.8MB/s] 46% 47.7M/104M [00:01<00:01, 32.6MB/s] 50% 52.3M/104M [00:01<00:01, 34.2MB/s] 54% 56.7M/104M [00:01<00:01, 36.4MB/s] 59% 61.8M/104M [00:01<00:01, 40.2MB/s] 64% 67.2M/104M [00:01<00:00, 44.1MB/s] 69% 72.0M/104M [00:01<00:00, 43.5MB/s] 74% 76.9M/104M [00:01<00:00, 45.7MB/s] 79% 82.3M/104M [00:02<00:00, 48.5MB/s] 84% 87.2M/104M [00:02<00:00, 45.5MB/s] 89% 92.6M/104M [00:02<00:00, 48.3MB/s] 94% 98.0M/104M [00:02<00:00, 50.6MB/s] 99% 103M/104M [00:02<00:00, 46.7MB/s] 100% 104M/104M [00:02<00:00, 43.6MB/s]
[2021-12-22 17:29:20,949] {subprocess.py:89} INFO - Downloading: "https://github.com/TencentARC/GFPGAN/releases/download/v0.2.0/GFPGANCleanv1-NoCE-C2.pth" to /opt/conda/lib/python3.8/site-packages/gfpgan/weights/GFPGANCleanv1-NoCE-C2.pth
[2021-12-22 17:29:20,950] {subprocess.py:89} INFO - 
[2021-12-22 17:29:29,000] {subprocess.py:89} INFO -   0% 0.00/332M [00:00<?, ?B/s]  0% 216k/332M [00:00<02:38, 2.20MB/s]  0% 1.05M/332M [00:00<02:02, 2.84MB/s]  1% 3.26M/332M [00:00<01:29, 3.84MB/s]  2% 7.91M/332M [00:00<01:04, 5.24MB/s]  4% 12.8M/332M [00:00<00:46, 7.18MB/s]  5% 16.8M/332M [00:00<00:34, 9.54MB/s]  6% 20.7M/332M [00:00<00:26, 12.4MB/s]  7% 24.7M/332M [00:00<00:20, 15.7MB/s]  9% 28.8M/332M [00:00<00:16, 19.4MB/s] 10% 32.8M/332M [00:01<00:13, 23.1MB/s] 11% 36.8M/332M [00:01<00:11, 26.7MB/s] 12% 41.0M/332M [00:01<00:10, 30.2MB/s] 14% 45.0M/332M [00:01<00:09, 33.0MB/s] 15% 49.2M/332M [00:01<00:08, 35.7MB/s] 16% 53.4M/332M [00:01<00:07, 37.9MB/s] 17% 57.5M/332M [00:01<00:07, 39.3MB/s] 19% 61.9M/332M [00:01<00:06, 41.0MB/s] 20% 66.0M/332M [00:01<00:06, 41.5MB/s] 21% 70.4M/332M [00:01<00:06, 42.6MB/s] 22% 74.6M/332M [00:02<00:06, 42.9MB/s] 24% 78.9M/332M [00:02<00:06, 43.6MB/s] 25% 83.2M/332M [00:02<00:05, 44.1MB/s] 26% 87.6M/332M [00:02<00:05, 44.4MB/s] 28% 92.0M/332M [00:02<00:05, 45.1MB/s] 29% 96.4M/332M [00:02<00:05, 44.9MB/s] 30% 101M/332M [00:02<00:05, 45.4MB/s]  32% 105M/332M [00:02<00:05, 45.5MB/s] 33% 110M/332M [00:02<00:05, 45.7MB/s] 34% 114M/332M [00:02<00:05, 45.7MB/s] 36% 118M/332M [00:03<00:04, 46.0MB/s] 37% 123M/332M [00:03<00:04, 45.7MB/s] 38% 127M/332M [00:03<00:04, 46.5MB/s] 40% 132M/332M [00:03<00:04, 46.6MB/s] 41% 136M/332M [00:03<00:04, 46.5MB/s] 42% 141M/332M [00:03<00:04, 47.2MB/s] 44% 146M/332M [00:03<00:04, 46.8MB/s] 45% 150M/332M [00:03<00:04, 47.3MB/s] 47% 155M/332M [00:03<00:03, 47.0MB/s] 48% 159M/332M [00:03<00:03, 47.4MB/s] 49% 164M/332M [00:04<00:03, 47.5MB/s] 51% 168M/332M [00:04<00:03, 47.2MB/s] 52% 173M/332M [00:04<00:03, 47.8MB/s] 53% 178M/332M [00:04<00:03, 47.4MB/s] 55% 182M/332M [00:04<00:03, 48.0MB/s] 56% 187M/332M [00:04<00:03, 47.9MB/s] 58% 192M/332M [00:04<00:03, 47.6MB/s] 59% 196M/332M [00:04<00:02, 48.5MB/s] 60% 201M/332M [00:04<00:02, 48.2MB/s] 62% 206M/332M [00:04<00:02, 48.6MB/s] 63% 210M/332M [00:05<00:02, 48.4MB/s] 65% 215M/332M [00:05<00:02, 48.5MB/s] 66% 220M/332M [00:05<00:02, 48.8MB/s] 68% 224M/332M [00:05<00:02, 48.7MB/s] 69% 229M/332M [00:05<00:02, 49.0MB/s] 70% 234M/332M [00:05<00:02, 48.8MB/s] 72% 239M/332M [00:05<00:02, 48.8MB/s] 73% 243M/332M [00:05<00:01, 49.0MB/s] 75% 248M/332M [00:05<00:01, 48.9MB/s] 76% 253M/332M [00:05<00:01, 49.2MB/s] 77% 257M/332M [00:06<00:01, 49.1MB/s] 79% 262M/332M [00:06<00:01, 48.8MB/s] 80% 267M/332M [00:06<00:01, 49.2MB/s] 82% 272M/332M [00:06<00:01, 49.1MB/s] 83% 276M/332M [00:06<00:01, 49.2MB/s] 85% 281M/332M [00:06<00:01, 49.3MB/s] 86% 286M/332M [00:06<00:00, 49.2MB/s] 87% 291M/332M [00:06<00:00, 49.5MB/s] 89% 295M/332M [00:06<00:00, 49.2MB/s] 90% 300M/332M [00:06<00:00, 49.5MB/s] 92% 305M/332M [00:07<00:00, 49.3MB/s] 93% 310M/332M [00:07<00:00, 49.5MB/s] 95% 314M/332M [00:07<00:00, 49.1MB/s] 96% 319M/332M [00:07<00:00, 49.5MB/s] 97% 324M/332M [00:07<00:00, 49.2MB/s] 99% 329M/332M [00:07<00:00, 49.4MB/s]100% 332M/332M [00:07<00:00, 45.4MB/s]
[2021-12-22 17:29:29,159] {subprocess.py:89} INFO - Testing 0 2264
[2021-12-22 17:29:29,273] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/nn/functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details.
[2021-12-22 17:29:29,273] {subprocess.py:89} INFO -   warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "
[2021-12-22 17:29:33,933] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:29:33,946] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=super_resolution, execution_date=20211222T075518, start_date=20211222T082915, end_date=20211222T082933
[2021-12-22 17:29:33,981] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:29:34,005] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
